.gitignore

```diff
@@ -24,3 +24,5 @@ npm-debug.log
 .rubygems-gem-maven-plugin
 *.sublime-*
 .temp/
+
+*.iml
```

Comment:
why do we need this in this git project?
there is already an exclude pattern ".iml" - is this existing pattern not correct?

Reply:
As far as I can see, .iml does not match a file like Foo.iml as it requires a wildcard. Imho it's currently only a dot-file match. 
https://git-scm.com/book/en/Git-Basics-Recording-Changes-to-the-Repository#_ignoring

Reply:
if the previous setting is wrong we should correct it instead of adding a new one